README.md

```diff
@@ -71,10 +70,10 @@ We want:
 
 There are a great number of tests available to test a JVM, starting with the OpenJDK regression tests.  In addition to running the OpenJDK regression tests, we will increase the amount of testing and coverage by pulling in other open tests.  These new tests are not necessarily written using the jtreg format.
 
-Why the need for other testing?  The OpenJDK regression tests are a great start, but eventually you may want to be able to test how performant is your code, and whether some 3rd party applications still work.  We will begin to incorporate more types of testing, including:
+Why the need for another testing?  The OpenJDK regression tests are a great start, but eventually, you may want to be able to test how performant is your code, and whether some 3rd party applications still work.  We will begin to incorporate more types of testing, including:
```

Comment:
This change does not improve this sentence.  It is more correct to say "why the need for other testing" than it is to say "why the need for another testing".